changelog/upgrade-notes.md

```diff
@@ -1,5 +1,18 @@
 ## Upgrade notes
 
+### v3.13.0
+
+#### `proj4js` integration
+
+Before this release, OpenLayers depended on the global proj4 namespace. When using for instance browserify you might not want to depend on the global `proj4` namespace. You can use the `ol.proj.setProj4` function to set the proj4 function object. For example in a browserify ES6 environment:
```

Comment:
Maybe move (or copy) some of this (especially the code snippet) to the `ol.proj.setProj4` API docs.